pbxplore/tests/__init__.py

```diff
@@ -20,24 +20,24 @@ def module_info(module_name):
     try:
         module = importlib.import_module(module_name)
         if hasattr(module, '__version__'):
-            print("{0} version {1}".format(module_name, module.__version__))
+            print(("{0} version {1}".format(module_name, module.__version__)))
```

Comment:
Why the extra set of parentheses?

Reply:
2to3 script... I explained in a comment further